apps/dex/example/values.yaml

```diff
@@ -1,10 +1,9 @@
 dex:
-  dex:
```

Comment:
Please revert changes in this file. It was correct - we need to reference dependency of dependency in the example chart. This updated version is failing.

Reply:
File reverted.